example/vite/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "graz-vite-example",
```

Comment:
rename this to `@project/example-vite`
and this https://github.com/strangelove-ventures/graz/pull/88/files#diff-1f7effcfbe4fea3b76c7f3a88e9c48fae3a0351d8373c909b09bb9400252e84dR2 to `@project/example-next`